boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,80 @@
+# fe-angular
+
+Angular quick starter project
+
+ ## Purpose of this quickstarter
+ 
+ This quick starter gives you a boilerplate code for new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift. 
+
+ ## What files / architecture is generated?
+ ```
+.
+├── Jenkinsfile
+├── README.md
+├── angular.json
+├── docker
+│   ├── Dockerfile
+│   └── nginx.vh.default.conf.nginx
+├── e2e
+│   ├── protractor.conf.js
+│   ├── src
+│   │   ├── app.e2e-spec.ts
+│   │   └── app.po.ts
+│   └── tsconfig.e2e.json
+├── package.json
+├── sonar-project.properties
+├── src
+│   ├── app
+│   │   ├── app.component.html
+│   │   ├── app.component.scss
+│   │   ├── app.component.spec.ts
+│   │   ├── app.component.ts
+│   │   └── app.module.ts
+│   ├── assets
+│   ├── browserslist
+│   ├── environments
+│   │   ├── environment.prod.ts
+│   │   └── environment.ts
+│   ├── favicon.ico
+│   ├── index.html
+│   ├── karma.conf.js
+│   ├── main.ts
+│   ├── polyfills.ts
+│   ├── styles.scss
+│   ├── test.ts
+│   ├── tsconfig.app.json
+│   ├── tsconfig.spec.json
+│   └── tslint.json
+├── tsconfig.json
+└── tslint.json
+
+7 directories, 31 files
+```
+
+ ## Frameworks used
+
+This project is generated by Angular CLI. (https://cli.angular.io/)
+ * Angular CLI version `6.0.7`
+ * Angular version `^6.0.3`
+ * Typescript version - `2.8.1`
```

Comment:
How will those version be kept up-to-date? I would probably remove them ...

Reply:
Done!